node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java

```diff
@@ -42,20 +42,30 @@ public boolean converge(TaskContext context) {
         StringBuilder diff = new StringBuilder();
         boolean modified = false;
 
-        for (String line : lines) {
+        for (int i = 0; i < lines.size(); i++) {
+            String line = lines.get(i);
             LineEdit edit = editor.edit(line);
             switch (edit.getType()) {
                 case REMOVE:
                     modified = true;
                     maybeRemove(diff, line);
                     break;
-                case REPLACE:
+                case REPLACE: {
                     modified = true;
                     String replacementLine = edit.replacementLine();
                     newLines.add(replacementLine);
                     maybeRemove(diff, line);
                     maybeAdd(diff, replacementLine);
                     break;
+                }
+                case INSERT: {
+                    modified = true;
+                    String replacementLine = edit.replacementLine();
+                    newLines.add(replacementLine);
+                    maybeAdd(diff, replacementLine);
+                    i--;
```

Comment:
This should work but forces the caller to keep some state to avoid making the same edit ad-infinitum. An alternative is to:
 - Make all edits to the current line (REPLACE, REMOVE, NONE) also allow the insertion of any number of lines prior to the currently edited line.

Reply:
Fixed, PTAL.